oc/openshift_ops.py

```diff
@@ -25,6 +25,38 @@ def __init__(self):
         self.v1_pods = dyn_client.resources.get(
             api_version='v1', kind='Pod'
         )
+        self.v1_deployments = dyn_client.resources.get(
+            api_version='v1', kind='Deployment'
+        )
+        self.v1_services = dyn_client.resources.get(
```

Comment:
Can we remove the v1 from the variable name?
The api version is v1 but the name shouldn't mention that

+ can we maintain the 'v1' in the defaults so as soon as the api version with change, we will be able to change only 1 parameter?

Reply:
Done.